code/particle/util/ParticleProperties.cpp

```diff
@@ -6,7 +6,7 @@
 
 namespace particle {
 namespace util {
-ParticleProperties::ParticleProperties() : m_radius(0.0f, 1.0f), m_lifetime(0.0f, 1.0f) {
+ParticleProperties::ParticleProperties() : m_radius(0.0f, 1.0f), m_lifetime(0.0f, 1.0f), m_length (0.0f, 1.0f){
```

Comment:
Since exiting code should be unaffected, the random range should only contain the single, "magic" zero value.
```.suggestion
ParticleProperties::ParticleProperties() : m_radius(0.0f, 1.0f), m_lifetime(0.0f, 1.0f), m_length (0.0f) {
```